docs/core/building/index.md

```diff
@@ -0,0 +1,107 @@
+
+```bat
+    .\build skiptests 
+```
+
+which will build the Debug flavor which has development time checks (asserts), or 
+
+```bat 
+    .\build release skiptests
+```
+
+to build the release (full speed) flavor.  You can find more build options with build by using the -? or -help qualifier.   
+
+#### Using Your Build
+
+The build places all of its generated files under the `bin` directory at the base of the repository.
+There is a `bin\Log` directory that contains log files generated during the build (Most useful when the build fails).
+The actual output is placed in a `bin\Product\[platform].[CPU architecture].[build type]` directory, such as `bin\Product\Windows_NT.x64.Release`.
+
+While the 'raw' output of the build is sometimes useful, normally you are only interested in the Nuget packages, which are placed in the `.nuget\pkg` subdirectory of the above output directory.
+
+There are two basic techniques for using your new runtime.
+
+ 1. **Use dotnet.exe and Nuget to compose an application**.
+    See [Using Your Build](https://github.com/dotnet/coreclr/blob/master/Documentation/workflow/UsingYourBuild.md) for instructions on creating a program that uses your new runtime by using the NuGet packages you just created and the'dotnet' command line interface.  This is the expected way non-runtime developers are likely to consume your new runtime.    
+
+ 2. **Use corerun.exe to run an application using unpackaged Dlls**.
+    This repository also defines a simple host called corerun.exe that does NOT take any dependency on NuGet.
+    This host has to be told where to get the necessary DLLs you actually use, and you have to manually gather them together.
+    This is the technique that all the tests in the repo use, and is useful for quick local 'edit-compile-debug' loop (e.g. preliminary unit testing).
+    See [Executing .NET Core Apps with CoreRun.exe](https://github.com/dotnet/coreclr/blob/master/Documentation/workflow/UsingCoreRun.md) for details on using this technique.
+
+### Building the CLI
+
+The source code for the .NET Core CLI can be found in [the `dotnet/cli` repository on GitHub](https://github.com/dotnet/cli/).
+
+#### Prerequisites
+
+In order to build .NET Command Line Interface, you need the following installed on you machine.
+
+* Windows & Linux:
+    - git on the PATH
+* macOS:
+    - git on the PATH
+    - Xcode
+    - OpenSSL
+
+#### building the CLI
+
+In order to build, run `build.cmd` or `build.sh` from the root depending on your OS. If you don't want to execute tests, run `build.cmd /t:Compile` or `./build.sh /t:Compile`. To build the CLI in macOS Sierra, you need to set the DOTNET_RUNTIME_ID environment variable by running `export DOTNET_RUNTIME_ID=osx.10.11-x64`.
+
+#### Using your build
+
+Use `artifacts/{RID}/stage2/dotnet` to try out the `dotnet` command. You can also add `artifacts/{os}-{arch}/stage2` to the PATH if you want to use the build output when invoking `dotnet` from the current console.
```

Comment:
AFAIK, this is the actual path to the dotnet executable, except `{RID}` is a placeholder depends on your OS (including the specific version of a specific Linux distro) and architecture.